docs/development/README.md

```diff
@@ -46,9 +46,25 @@ ENABLE_SENTRY=
 
 ### PostgreSQL 12
 
-If you are installing a local instance of PostgreSQL on Ubuntu, see the steps in the separate [guide.](./UBUNTU2004-PG12.md)
+#### Setup the database with docker
+
+Run `sudo docker-compose up --no-start` to create the container.
```

Comment:
Do we need to run this all as root ? (On OSX at least it ran ok without)

Reply:
Currently, yes. Rootless mode is available but is experimental and limited.